src/domain/journey/subspace/layout/SubspaceDialog.ts

```diff
@@ -9,4 +9,5 @@ export enum SubspaceDialog {
   Share = 'Share',
   ManageFlow = 'ManageFlow',
   Settings = 'Settings',
+  Updates = 'dashboard/updates',
```

Comment:
It's easy to plug in because of the current implementation in Spaces. We'll need to distinguish the level otherwise adding more logic. I'll look into it.